src/components/capture/Create.tsx

```diff
@@ -301,7 +307,7 @@ function CaptureCreate() {
                 supabaseClient
                     .from(TABLES.DRAFTS)
                     .insert({
-                        detail: entityName,
+                        detail: `${entityPrefix.title}${entityName}`,
```

Comment:
Since this concatenation is performed in more than one place, would it be beneficial to save it off in a constant?

If you plan to replace this code fragment soon and in a fashion that renders the constant unnecessary, there is no reason to change the logic. If the code fragment will linger for awhile, it may be worth considering.